README.md

```diff
@@ -1,13 +1,58 @@
 # cQASM language specification
 
-This repository contains the specification documents of the cQASM quantum programming language. 
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-_The specification hosted through GitHub Pages under the following link:_ 
+This repository contains the language specification of the cQASM quantum programming language.
+cQASM stands for *c*ommon *Q*uantum *AS*se*M*bly language.
 
-### [<img src="./docs/_static/book.svg"> cQASM language specification](https://qutech-delft.github.io/cQASM-spec/) 
+## Deployment
```

Comment:
The Deployment section should indeed be a page confluence. So it should be removed from the README.

As mentioned in a previous comment, the following section can be added to the README, for use by potential contributors:

## Generate the cQASM language specification locally

Clone the repository to your local machine

```shell
git clone https://github.com/QuTech-Delft/cQASM-spec.git
```

We recommend to create a virtual environment (_e.g._, [venv](https://docs.python.org/3/library/venv.html)), run it, and install the requirements

```shell
pip install -r requirements.txt
```

Once the required dependencies have installed, the language specification can be served locally at <http://localhost:8000> by running:

```shell
mkdocs serve
```

The language specification can also be viewed via a provisioned Docker container; no virtual environment is required in this case. Run the Docker container with the following command

```shell
docker compose up -d
```

The language specification can now be viewed at <http://localhost:8106>.

Reply:
Done. With:
- some minor tweaks (e.g., adding colons at the end of a phrase), and
- changing the title to `How to generate...`.

Reply:
I'm not a fan of the ellipsis (`...`), it could be `How to generate` or `Generate the specification locally`